test-e2e/panels/structures.test.ts

```diff
@@ -465,9 +464,16 @@ test('molecules 1H spectrum', async ({ page, browserName }) => {
     await expect(nmrium.page.locator('#molSVG')).toBeHidden();
     // Check selected molecule.
 
-    await expect(
-      moleculePanelLocator.locator('p:has-text("1 / 2"), p:has-text("1 / 3")'),
-    ).toBeVisible();
+    if (browserName === 'chromium') {
+      await expect(
+        nmrium.page.locator('_react=MoleculePanel >> text=1 / 3'),
+      ).toBeVisible();
+    } else {
+      await expect(
+        nmrium.page.locator('_react=MoleculePanel >> text=1 / 2'),
+      ).toBeVisible();
+    }
```

Comment:
Why is there a different result on different browsers?

Reply:
It was like that before I just improved the test. 
Maybe @hamed-musallam knows why it's diffrent.

Reply:
At that time, the copy/paste molecule structure function worked well only in Chrome, but this is no longer true.

This feature needs to be tested again, and then these lines should be refactored from the test code

Reply:
Still working only on Chrome. As you can see in test only chrome have 3 molecules.

Reply:
ok let's leave it like that